ui/app/AppLayouts/Profile/views/wallet/AccountView.qml

```diff
@@ -18,8 +17,10 @@ import "../../popups"
 Item {
     id: root
     signal goBack
+    signal copyToClipboard(string address)
 
     property WalletStore walletStore
+    property var currentAccount
```

Comment:
Here you add the `currentAccount` but you still keep the `walletStore` around

Reply:
Ok, so both are still needed; why not put the `currentAccount` in a private QtObject?

Reply:
This component is still too coupled with other components, so removing completly the walletStore at this stage is not possible as a deeper refactor is needed and we're not at that stage yet, this is a first pass at adding some of this to storybook